porter/v1/porter_app.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package porter.v1;
+
+import "porter/v1/service.proto";
+
+message PorterApp {
+  string name = 1;
+  map<string, Service> services = 2;
+  map<string, string> env = 3;
+  Build build = 4;
+  Service release = 5;
+  AppImage image = 6;
+}
+
+message Build {
+  string context = 1;
```

Comment:
dont forget to add comments which provide context as to what, and when each field is used @ianedwards